config/application.rb

```diff
@@ -10,7 +10,6 @@
 require 'action_view/railtie'
 require 'action_cable/engine'
 require 'active_storage/engine'
-require 'sprockets/railtie'
```

Comment:
Why are we doing this change?

Reply:
Because that would import sprockets 4.0.0, which has vulnerabilities .
You can read more on links from second item in description.

Reply:
How can `sprockets 4.0.0` be imported if you put `gem 'sprockets', '~> 3.7.2'` in the `Gemfile`?

Reply:
🤷‍♀ I don't know about that. It was just not taking `Gemfile` version.

Reply:
Maybe you were running `brakeman` without prefixing it with `bundle exec`? https://thoughtbot.com/blog/but-i-dont-want-to-bundle-exec

Reply:
I was running `code_analysis` task:
`task :code_analysis do
  sh 'bundle exec brakeman . -z -q'
...
end
`

Reply:
I tried it myself (putting back this line) and didn't get any warning message. Let me know if you want to pair on this one

Reply:
🤔 that is certainly weird, who knows what did I do 😆 
Sure! Let's do it